src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student2/BookController2.java

```diff
@@ -6,10 +6,10 @@
 
 @RestController
 public class BookController2 {
-    Book book = new Book();
 
     @GetMapping("student2/api/v1/book")
     public String getMessage() {
```

Comment:
We should return the Book class here, no reason to use `toString()` in this case.